### ISSUE TITLE
[Feature Request] Change the default source used with the "@bot Play [song title]"

### ISSUE THREAD

**Issue by froglicker** [jagrosh/MusicBot#1768]
### Is your feature request related to a problem? Please describe.

Youtube playback being largely borked.

### What is your ideal solution to the problem?

Enable users to choose a new default source for "@bot Play [song title]" from the supported options.

    YouTube
    SoundCloud
    Bandcamp
    Vimeo
    Twitch streams
    Local files
    HTTP URLs

### How would this feature be used?

via the config.txt or with another bot command

### Checklist

- [X] I have checked [the documentation](https://jmusicbot.com/) to make sure this feature doesn't already exist
- [X] I have searched for [similar feature requests](https://github.com/jagrosh/MusicBot/labels/enhancement)
- [X] I have confirmed that my requests is **not** on the [list of things that will not be added](https://jmusicbot.com/things-that-wont-be-added)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by Monke115x** [comment]
YES. Youtube doesnt work or I just dont know how to get it too but If I could just select soundcloud as my main that would be amazing. Also if the search could be updated to work with soundcloud if that was selected as the main option.

**Comment by othello777** [comment]
youtube works if you update the youtube lavaplayer maven dependencies to their latest versions. you might also need to use the oauth branch, unsure

**Comment by MichailiK** [comment]
> you might also need to use the oauth branch

do not use the oauth2 branch. i abandoned & deleted that branch a few weeks ago, because youtube patched the oauth2 workaround.
